### PR TITLE
Postgresql Cheat sheet: Added two new sections

### PR DESCRIPTION
**Pull request by mayank7**
Added two new sections namely PostgreSql commands and Role to Postgresql Cheat sheet.

IA Page: https://duck.co/ia/view/postgresql_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
## [PostgreSQL Cheat Sheet](https://duck.co/ia/view/postgresql_cheat_sheet)

**Description**: Overview of basic PostrgreSQL usage

**Example Query**: [postgresql cheat sheet](https://beta.duckduckgo.com/?q=postgresql cheat sheet), [](https://beta.duckduckgo.com/?q=)

**Tab Name**: Answer

**Source**:  

*These are the important fields from the IA page.  Please check these for errors or missing information and update the [IA page](https://duck.co/ia/view/postgresql_cheat_sheet)*

---
*This is an automated message which will be updated as changes are made to the [IA page](https://duck.co/ia/view/postgresql_cheat_sheet)*

File: share/goodie/cheat_sheets/json/postgresql.json [hunk 1]
```diff
@@ -9,7 +9,9 @@
   },
   "section_order": [
     "Installation (Debian)",
+    "PostgreSql Commands",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/postgresql.json]: Can you format this as `PostgreSQL` ?

**Comment by mayank7** [comment]
@mintsoft I formatted PostgreSql Commands to PostgreSQL as suggested by you.

File: share/goodie/cheat_sheets/json/postgresql.json [hunk 2]
```diff
@@ -3,13 +3,16 @@
   "name": "PostgreSQL",
   "description": "PostgreSQL is an object-relational database management system",
   "template_type": "terminal",
+  "aliases": ["postgresql"],
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/postgresql.json]: It's redundant to add the same alias as the name of the sheet, can you remove this?

Comment (mayank7) [inline share/goodie/cheat_sheets/json/postgresql.json]: @mintsoft I removed the alias and did the modification suggested by you.

**Pull request merged**